web_responsive/models/res_users.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    show_apps_menu_on_load = fields.Selection(
+        selection=[
+            ("default", "Never (default)"),
```

Comment:
```suggestion
            ("never", "Never (default)"),
```

Default value in db is not explicit.